modules/nf-core/polypolish/polish/meta.yml

```
@@ -0,0 +1,69 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "polypolish"
+description: Polishing genome assemblies with short reads.
+keywords:
+  - assembly polishing
+  - genome polishing
+  - ont
+tools:
+  - "polypolish":
+      description: "Polishing genome assemblies with short reads."
+      homepage: "https://github.com/rrwick/Polypolish"
+      documentation: "https://github.com/rrwick/Polypolish/wiki"
+      tool_dev_url: "https://github.com/rrwick/Polypolish"
+      doi: "https://doi.org/10.1099/mgen.0.001254"
+      licence:
+      identifier: biotools:polypolish
+
+input:
+  # Only when we have meta
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+
+    - "*.fasta":
```

```suggestion
    - fasta:
```

It's the output channels where you use the root as the pattern